### ISSUE TITLE
Negative transfer problem

### ISSUE THREAD

**Issue by donggd** [FengZhu-Joey/GA-DTCDR#8]
Hello, when I used your model in other graph network data sets, negative transfer（That is, single domain effect is better than double） occurred. How did you solve the problem of negative transfer? Or can you offer some advice? thanks!

**Comment by FengZhu-Joey** [comment]
Thanks for your attention to our work. As for your issue, my suggestions can be summarized as follows: (1) you can design an end-to-end model which can ease the problem of the objective distortion caused by two-step training processes; (2) if you use a joint loss function for the two or more domains, you can attempt to use some separated loss for each domain with a separated optimizer, which can ease the problem of the seesaw phenomenon; (3) to directly ease the problem of the negative transfer, you'd better design a suitable mask for the attention network rather than a fully-connected attention network (this is because transferring the knowledge of some common entities learned from different domains can negatively affect our model). I hope these suggestions may help you.

**Comment by donggd** [comment]
Thanks! Your suggestions help me a lot!
